ui/components/component-library/avatar-icon/avatar-icon.test.js

```diff
@@ -100,7 +100,9 @@ describe('AvatarIcon', () => {
       />,
     );
 
-    expect(getByTestId('success')).toHaveClass('box--color-success-default');
+    expect(getByTestId('success')).toHaveClass(
+      'mm-text--color-success-default',
+    );
```

Comment:
We shouldn't have to update this the text component should use the `Box` component `color` prop have created an issue here https://github.com/MetaMask/metamask-extension/issues/18209 cc @garrettbear